cinema/serializers.py

```diff
@@ -125,10 +125,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
+            ValidationError,
         )
```

Comment:
Ensure that the `validate_ticket` method in the `Ticket` model is implemented correctly. This method is called during validation to check the validity of the ticket's row, seat, and movie session. If it's not implemented or has issues, it could lead to validation errors.